starter-code/pizza.js

```diff
@@ -64,22 +92,86 @@ function renderButtons() {
 }
 
 function renderPrice() {
-  // Iteration 4: change the HTML of `<aside class="panel price">`
+  let totalPrice = 10;
+  let stringFinished = "";
+  if (state.pepperonni) {
+    stringFinished += "<li>$1 pepperonni</li>";
+    totalPrice += ingredients.pepperonni.price;
+  }
+  if (state.mushrooms) {
+    stringFinished += "<li>$1 mushrooms</li>";
+    totalPrice += ingredients.mushrooms.price;
+  }
+  if (state.greenPeppers) {
+    stringFinished += "<li>$1 green peppers</li>";
+    totalPrice += ingredients.greenPeppers.price;
+  }
+  if (state.whiteSauce) {
+    stringFinished += "<li>$3 white sauce</li>";
+    totalPrice += ingredients.whiteSauce.price;
+  }
+  if (state.glutenFreeCrust) {
+    stringFinished += "<li>$5 gluten-free crust</li>";
+    totalPrice += ingredients.glutenFreeCrust.price;
+  }
+
+  document.querySelector("aside > ul").innerHTML = stringFinished;
+  document.querySelector("aside > strong").innerHTML = "$" + totalPrice;
 }
 
-
-renderEverything()
+// Iteration 4: change the HTML of `<aside class="panel price">`
+/*let stringFinished = "";
+  if (state.pepperonni) {
+    stringFinished = document.createElement(<li>$1 pepperonni</li>);
+  }
+  if (state.mushrooms) {
+    stringFinished = document.createElement(<li>$1 mushrooms</li>);
+  }
+  if (state.greenPeppers) {
+    stringFinished = document.createElement(<li>$1 green peppers</li>);
+  }
+  if (state.whiteSauce) {
+    stringFinished = document.createElement(<li>$3 white sauce</li>);
+  }
+  if (state.glutenFreeCrust) {
+    stringFinished = document.createElement(<li>$5 gluten-free</li>);
+  }
+  document.querySelector("body > aside > ul").innerHTML = stringFinished;
+  return stringFinished;
+}*/
+
+renderEverything();
 
 // Iteration 1: Example of a click event listener on `<button class="btn btn-pepperonni">`
-document.querySelector('.btn.btn-pepperonni').onclick = function() {
-  state.pepperonni = !state.pepperonni
-  renderEverything()
```

Comment:
All pretty decent!
You have missed to do the toggle function for each button 
Something like this ``document.querySelector(".btn.btn-sauce").onclick = function() {
  event.target.classList.toggle("active");}``

Reply:
so you should only add to each onclick function ``event.target.classList.toggle("active")``